src/views/chatView/utils/ContextUtils.ts

```
@@ -0,0 +1,231 @@
+
+  let globalConfig = await lsClient.sendLsClientRequest<ZenmlGlobalConfigResp>('getGlobalConfig');
+  let apiToken = globalConfig.store.api_token;
+  let dashboardUrl = globalConfig.store.url;
+
+  if (!apiToken) {
+    throw new Error('API Token is not available in global configuration');
+  }
+
+  let pipelineRunSteps = await getPipelineRunNodes('step');
+
+  if (pipelineRunSteps && pipelineRunSteps[0]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)